.github/workflows/experimental-release.yaml

```diff
@@ -0,0 +1,86 @@
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Environment setup
+        uses: ./.github/actions/setup
+
+      - name: Login to GHCR
+        run: uds run registry-login --set REGISTRY=ghcr.io --set REGISTRY_USERNAME=dummy --set REGISTRY_PASSWORD=${{ secrets.GITHUB_TOKEN }} --set REGISTRY_RETRY_INTERVAL=90
+
+      - name: Login to registry1
+        run: uds run registry-login --set REGISTRY=registry1.dso.mil --set REGISTRY_USERNAME=${{ secrets.IRON_BANK_ROBOT_USERNAME }} --set REGISTRY_PASSWORD=${{ secrets.IRON_BANK_ROBOT_PASSWORD }} --set REGISTRY_RETRY_INTERVAL=90
+
+      - name: Set versions to experimental
+        run: |
+          yq -ei '.metadata.version="experimental", (.packages[]|select(has("ref"))|select(.name=="core")).ref="experimental"' bundles/k3d-standard/uds-bundle.yaml
+          yq -ei '.metadata.version="experimental", (.packages[]|select(has("ref"))|select(.name=="core-slim-dev")).ref="experimental"' bundles/k3d-slim-dev/uds-bundle.yaml
+          yq -ei '.metadata.version="experimental"' packages/slim-dev/zarf.yaml
+          yq -ei '.metadata.version="experimental"' packages/standard/zarf.yaml
+
+      - name: Create Packages and Bundles
+        run: |
+
+          ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml standard-package --no-progress --set FLAVOR=${{ matrix.flavor }}
+          ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml k3d-standard-bundle --no-progress
+          ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml slim-dev-package --no-progress --set FLAVOR=${{ matrix.flavor }}
+          ZARF_ARCHITECTURE=amd64 uds run -f tasks/create.yaml k3d-slim-dev-bundle --no-progress
+
+          if [ "${{ matrix.flavor }}" != "registry1" ]; then
+            ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml standard-package --no-progress --set FLAVOR=${{ matrix.flavor }}
+            ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml k3d-standard-bundle --no-progress
+            ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml slim-dev-package --no-progress --set FLAVOR=${{ matrix.flavor }}
+            ZARF_ARCHITECTURE=arm64 uds run -f tasks/create.yaml k3d-slim-dev-bundle --no-progress
+          fi
+
+      # Standard Package by default tests what's in the Istio Package
+      #- name: Test amd64 Bundle
+      #  run: |
+      #    uds run -f tasks/deploy.yaml k3d-standard-bundle --no-progress
+      #    uds run -f tasks/test.yaml validate-packages --no-progress
+
+      - name: Publish Standard Package
+        run: uds run -f tasks/publish.yaml standard-package --set FLAVOR=${{ matrix.flavor }} --set VERSION=experimental
+
+      - name: Publish Upstream Flavored Bundles
+        if: ${{ matrix.flavor != 'registry1'}}
+        run: uds run -f tasks/publish.yaml bundles --set VERSION=experimental
```

Comment:
DRY is good!  updated to extract the `publish` pieces of the workflow and based on the `snapshot` input provided by the calling workflows (either typical or snapshot), then it will release accordingly.